script/crossbinary

```diff
@@ -1,6 +1,29 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
NixOS for example \o/ (where `/bin/bash` doesn' exists) but `/usr/bin/env` exists (this is almost the *one thing* that is almost always at the same place).